third_party/move/tools/move-formatter/tests/aptos_framework_case/coin.fmt.move

```diff
@@ -0,0 +1,1443 @@
+
+    /// A holder of a specific coin types and associated event handles.
+    /// These are kept in a single resource to ensure locality of data.
+    struct CoinStore<phantom CoinType> has key {
+        coin: Coin<CoinType>,
+        frozen: bool,
+        deposit_events: EventHandle<DepositEvent>,
+        withdraw_events: EventHandle<WithdrawEvent>,
+    }
+
+    /// Maximum possible coin supply.
+    const MAX_U128: u128 = 340282366920938463463374607431768211455;
+
+    /// Configuration that controls the behavior of total coin supply. If the field
+    /// is set, coin creators are allowed to upgrade to parallelizable implementations.
+    struct SupplyConfig has key {
+        allow_upgrades: bool,
+    }
+
+    /// Information about a specific coin type. Stored on the creator of the coin's account.
+    struct CoinInfo<phantom CoinType> has key {
+        name: string::String,
+        /// Symbol of the coin, usually a shorter version of the name.
+        /// For example, Singapore Dollar is SGD.
+        symbol: string::String,
+        /// Number of decimals used to get its user representation.
+        /// For example, if `decimals` equals `2`, a balance of `505` coins should
+        /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
+        decimals: u8,
+        /// Amount of this coin type in existence.
+        supply: Option<OptionalAggregator>,
+    }
+
+    /// Event emitted when some amount of a coin is deposited into an account.
+    struct DepositEvent has drop, store {
+        amount: u64,
+    }
+
+    #[event]
+    struct Deposit<phantom CoinType> has drop, store {
+        account: address,
+        amount: u64,
+    }
+
+    /// Event emitted when some amount of a coin is withdrawn from an account.
+    struct WithdrawEvent has drop, store {
+        amount: u64,
+    }
+
+    #[event]
+    struct Withdraw<phantom CoinType> has drop, store {
+        account: address,
+        amount: u64,
+    }
+
+    /// Capability required to mint coins.
+    struct MintCapability<phantom CoinType> has copy, store {}
+
+    /// Capability required to freeze a coin store.
+    struct FreezeCapability<phantom CoinType> has copy, store {}
+
+    /// Capability required to burn coins.
+    struct BurnCapability<phantom CoinType> has copy, store {}
+
+    //
+    // Total supply config
+    //
+
+    /// Publishes supply configuration. Initially, upgrading is not allowed.
+    public(friend) fun initialize_supply_config(aptos_framework: &signer) {
+        system_addresses::assert_aptos_framework(aptos_framework);
+        move_to(
+            aptos_framework,
+            SupplyConfig {allow_upgrades: false}
+        );
+    }
+
+    /// This should be called by on-chain governance to update the config and allow
+    /// or disallow upgradability of total supply.
+    public fun allow_supply_upgrades(
+        aptos_framework: &signer,
+        allowed: bool
+    )
+        acquires SupplyConfig {
```

Comment:
is it a normal case that `acquires` should be in a separate line? cc @vineethk

Reply:
I think this may require a bit more discussion. Given that we are expanding to `reads` and `writes` along with `acquires`, the idea was that we separate all these entities out, otherwise it is hard to parse them (eg., you can say `reads A !writes B reads C, E !writes D`.

So, perhaps if there are more than one kind of resource specifier (`reads`, `writes`, `acquires`), then we need to have them out separated, else not (especially if they fit within column limit)?

I wanted to report this as a bug, but maybe I'll mention it here - we seem to have an issue where you can specify a very long list of resources after `acquires` (like `acquires LongName1, LongName2, ...`) and they all stay in the same line even if they exceed the column limit.

Reply:
In any case, I agree that `acquires` here would be better placed in the previous line.